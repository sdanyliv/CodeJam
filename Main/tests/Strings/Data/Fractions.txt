1.011.02
1.010.12
1.009.02
1.009.20
1.009.10
1.002.08
1.002.03
1.002.01