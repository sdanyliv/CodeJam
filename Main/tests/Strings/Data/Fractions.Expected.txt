1.002.01
1.002.03
1.002.08
1.009.02
1.009.10
1.009.20
1.010.12
1.011.02